.github/workflows/aws-e2e.yaml

```diff
@@ -0,0 +1,92 @@
+---
+name: Test nitric run
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+    types: [labeled]
+
+env:
+  GOPROXY: https://proxy.golang.org
+  AWS_REGION: us-east-1
+
+jobs:
+  e2e-aws:
+    if: ${{ github.event.label.name == 'e2e-aws' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: cli
+
+      - name: checkout the test app
+        uses: actions/checkout@v3
+        with:
+          repository: asalkeld/test-app
```

Comment:
might be good to transfer this into the nitrictech repository.

Reply:
done, might need approval from you or Jye